test/e2e/README.md

```diff
@@ -104,12 +104,15 @@ graph TD;
     A1--linux/amd64-->A2[oci image]
     A1--linux/arm64-->A3[oci image]
     A1--linux/arm/v7-->A4[oci image]
+    A1--linux/s390x-->A5[oci image]
```

Comment:
Changes in this file should be undo. Since this section provides a description for e2e test image (in `test\e2e\testdata\distribution\mount\images.tar.gz`), not oras released image in GHCR.

Reply:
I reverted this file to the commit before my changes and signed off both commits.